website/versioned_docs/version-0.20/more/css.mdx

```diff
@@ -23,6 +23,7 @@ projects. A few are given below:
 #### Styling Solutions
 
 -   [stylist](https://github.com/futursolo/stylist-rs) - A CSS-in-Rust styling solution for WebAssembly Applications.
+-   [tailwind-css](https://tailwindcss.com/docs/installation/play-cdn) - Tailwind Utility Classes.
```

Comment:
The tailwind CLI is a better way to use tailwind. A Trunk [hook can be used](https://github.com/yewstack/yew-playground/blob/ec03c4bb426b75658ec6d2482dbcec9d21786697/frontend/Trunk.toml#L5-L8) to build the CSS which is then [linked in index.html](https://github.com/yewstack/yew-playground/blob/ec03c4bb426b75658ec6d2482dbcec9d21786697/frontend/index.html#L7).